system-addon/lib/SectionsManager.jsm

```diff
@@ -66,7 +66,7 @@ const BUILT_IN_SECTIONS = {
     icon: "highlights",
     title: {id: "header_highlights"},
     maxRows: 3,
-    availableContextMenuOptions: ["CheckBookmark", "SaveToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl", "DeleteUrl"],
+    availableContextMenuOptions: ["CheckBookmarkOrArchive", "CheckSavedToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl", "CheckDeleteHistoryOrEmpty"],
```

Comment:
We want these "Check" versions for `topstories.availableContextMenuOptions` too for the in-page update. (Fortunately, topstories already doesn't have "DeleteUrl" :p)